strongbox-resources/strongbox-storage-resources/strongbox-storage-api-resources/src/main/resources/etc/conf/strongbox.xml

```diff
@@ -30,6 +30,17 @@
                 <!-- Secured repositories do not allow anonymous access -->
                 <repository id="releases" policy="release" implementation="file-system" layout="Maven 2" type="hosted" allows-force-deletion="true" checksum-headers-enabled="true" indexing-enabled="true"/>
                 <repository id="snapshots" policy="snapshot" implementation="file-system" layout="Maven 2" type="hosted" secured="true" checksum-headers-enabled="true"/>
+                <repository id="redeployment-validated-only"
```

Comment:
This particular test repository should not be stored in the `strongbox.xml` file. Clearly, in real-life configurations, such will configurations be here. (Actually, previously, there was a bunch of such test repositories defined here and I had to clean it up with a big refactoring).

Any test that needs a repository, should:
* Create a repository via test code
* Configure the repository as requires programatically 
* Generate the required artifacts in this repository
* Clean up the artifacts before the next run of the tests (for the cases where you need to execute the tests via an IDE)